gef.py

```diff
@@ -7613,7 +7613,12 @@ def context_code(self) -> None:
         cur_insn_color = gef.config["theme.disassemble_current_instruction"]
         pc = gef.arch.pc
         breakpoints = gdb.breakpoints() or []
-        bp_locations = [b.location for b in breakpoints if b.location and b.location.startswith("*")]
+        # breakpoint.locations was introduced in gdb 13.1
+        if len(breakpoints) and hasattr(breakpoints[-1], "locations"):
```

Comment:
If you're certain about the GDB version it was introduced, this check is simpler and more explicit (and you can remove the comments).

```suggestion
        if GDB_VERSION >= (13, 1):
```

Reply:
I think hasattr is safer, just in case the feature is backported or something in some distro.

Reply:
I think it's a lot less expressive about the meaning of the check (I suppose that's why the need of adding comments).

Reply:
Right, but that's probably worth the trade off, if it means that 1. We don't crash if we find some GDB version 13.1 or newer that somehow doesn't have this patch and 2. We can use this feature in versions that have the patch backported.